src/test/java/org/jabref/migrations/SpecialFieldsToSeparateFieldsTest.java

```diff
@@ -29,41 +39,40 @@ public void migrateToCorrectField(SpecialField field, String fieldInKeyword, Bib
 
     @Test
     public void noKewordToMigrate() {
-        BibEntry entry = new BibEntry().withField(StandardField.AUTHOR, "JabRef")
-                                       .withField(StandardField.KEYWORDS, "tdd");
-        BibEntry expected = new BibEntry().withField(StandardField.AUTHOR, "JabRef")
-                                          .withField(StandardField.KEYWORDS, "tdd");
+        entry.setField(StandardField.AUTHOR, "JabRef");
+        entry.setField(StandardField.KEYWORDS, "tdd");
+        expectedEntry.setField(StandardField.AUTHOR, "JabRef");
+        expectedEntry.setField(StandardField.KEYWORDS, "tdd");
```

Comment:
Please keep creation of the BibEntry in the test method here and use the fluid api to set the fields.